### PR TITLE
fea(api): Added API to find the configuration of pollers

### PR DESCRIPTION
**Pull request by callapa**
## Description

Added API to find the configuration of pollers

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [x] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 2.8.x
- [ ] 18.10.x
- [ ] 19.04.x
- [ ] 19.10.x
- [x] 20.04.x (master)

<h2> How this pull request can be tested ? </h2>

Consult the API documentation to see the urls to be tested.

#### Community contributors & Centreon team

- [x] I followed the **coding style guidelines** provided by Centreon
- [x] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [x] I have commented my code, especially **hard-to-understand areas** of the PR.
- [x] I have made corresponding changes to the **documentation**.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

#### Centreon team only

- [ ] I have made sure that the **unit tests** related to the story are successful.
- [ ] I have made sure that **unit tests cover 80%** of the code written for the story.
- [ ] I have made sure that **acceptance tests** related to the story are successful (**local and CI**)

### CODE REVIEW

**Comment by cgagnaire** [comment]
As discussed, we may need to find a better wording for those classes.

File: src/Centreon/Application/Controller/Configuration/MonitoringServerController.php [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        $this->monitoringServerService = $monitoringServerService;
+    }
+
+    /**
+     * Entry point to find the last hosts acknowledgements.
+     *
+     * @IsGranted("ROLE_API_CONFIGURATION", message="You are not authorized to access this resource")
+     * @Rest\Get(
+     *     "/configuration/monitoring-servers",
+     *     condition="request.attributes.get('version.is_beta') == true",
```
Comment (jdelpierre) [inline src/Centreon/Application/Controller/Configuration/MonitoringServerController.php]: It's normal to use "==" and not "===" ?

File: src/Centreon/Infrastructure/MonitoringServer/MonitoringServerRepositoryRDB.php [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+     * @inheritDoc
+     */
+    public function findServers(): array
+    {
+        $this->sqlRequestTranslator->setConcordanceArray([
+            'id' => 'id',
+            'name' => 'name',
+            'is_localhost' => 'localhost',
+            'address' => 'ns_ip_address',
+            'is_activate' => 'ns_activate'
```
Comment (jdelpierre) [inline src/Centreon/Infrastructure/MonitoringServer/MonitoringServerRepositoryRDB.php]: what does the prefix "ns" mean?

Comment (jdelpierre) [reply src/Centreon/Infrastructure/MonitoringServer/MonitoringServerRepositoryRDB.php]: Ok I have the answer ;) nagios server

File: doc/API/centreon-api-v2.yaml [hunk 3]
```diff
@@ -43,6 +44,10 @@ tags:
       * Acknowledge a host or a service
       * Cancel an acknowledgement of host or service
   - name: Authentication
+  - name: MonitoringServer
+    description: |
+      This API allow to:
```
Comment (kduret) [inline doc/API/centreon-api-v2.yaml]:
```suggestion
      This API allows to:
```

File: src/Centreon/Application/Controller/Configuration/MonitoringServerController.php (hunk 1 above)
Comment (callapa) [reply src/Centreon/Application/Controller/Configuration/MonitoringServerController.php]: It's not a PHP condition

**Pull request closed without merging**